apollo-backend/src/test/java/io/logz/apollo/SingleRegionBlockerTest.java

```diff
@@ -123,23 +123,27 @@ public void testSingleRegionBlockerAllowsDeploymentsWithDifferentAvailability()
         Environment env2 = ModelsGenerator.createEnvironment(availabilityStaging, null);
         env2.setId(apolloTestClient.addEnvironment(env2).getId());
 
+        Environment env3 = ModelsGenerator.createEnvironment(availabilityStaging, null);
+        env3.setId(apolloTestClient.addEnvironment(env3).getId());
+
         ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env1), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
         ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env2), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
+        ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env3), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
 
         blocker = createAndSubmitBlocker(apolloTestAdminClient, BlockerTypeName.SINGLE_REGION, "{}", null, serviceToBeLimitToOneRegion, null, env1.getAvailability());
 
-        List<Environment> prodAndStagingEnvironments = Arrays.asList(env1, env2);
-
         List<Callable<MultiDeploymentResponseObject>> deploymentRequests = new ArrayList<>();
-        addDeploymentRequests(apolloTestClient, serviceToBeLimitToOneRegion, deployableVersion, prodAndStagingEnvironments, deploymentRequests);
+
+        deploymentRequests.add(() -> apolloTestClient.addDeployment(String.valueOf(env1.getId()), String.valueOf(serviceToBeLimitToOneRegion.getId()), deployableVersion.getId()));
+        deploymentRequests.add(() -> apolloTestClient.addDeployment(env2.getId() + "," + env3.getId(), String.valueOf(serviceToBeLimitToOneRegion.getId()), deployableVersion.getId()));
 
         List<Future<MultiDeploymentResponseObject>> futures = executorService.invokeAll(deploymentRequests);
 
         List<MultiDeploymentResponseObject> results = futures.stream()
                 .map(this::futureGetUnchecked)
                 .collect(Collectors.toList());
 
-        assertThat(getSuccessfulDeploymentResponseObjects(results).size()).isEqualTo(2);
+        assertThat(getSuccessfulDeploymentResponseObjects(results).size()).isEqualTo(3);
```

Comment:
did you check that it actually fails before adding your fix?

Reply:
@matvey-mtn Mistakenly I created the solution before the test. but I am sure that it is ok.

Reply:
I see
No need in null check
all good!

Reply:
@maorshap it's ok, I don't know a lot of people who work pure TDD :) 
you can temporarily comment our your changes or ignore availability in the blocker
just to see that it fails

Reply:
@matvey-mtn I was not clear enough.. That what I have done :). all good